Documentation/Contributing/CodingGuidelines.md

```diff
@@ -542,4 +532,4 @@ Prefer using DateTime.UtcNow unless you actually need the localized times (a leg
 
 
 ## Set also
```

Comment:
```suggestion
## See also
```

Reply:
Weird, I didn't intentionally change these two lines? Thanks for fixing!